### ISSUE TITLE
Please us smaller animated gifs tutorial/images/1v2x*.gif files

### ISSUE THREAD

**Issue by narc-Ontakac2** [gap-packages/hap#120]
The largest one has 20Mb and is thereby bigger than the rest of the (compressed) gap-package.

I have come across this while packaging the latest version for Debian when I checked for the cause of the increased size. 

Unfortunately it is not easy to fix. It needs using the asymptote language and then convert a l lot of individual images into an animated gif (the latter one being the easy part).

**Comment by grahamknockillaree** [comment]
Maybe I should remove these animated gifs from the distribution. I'll look into this next week. Thanks.

**Comment by narc-Ontakac2** [comment]
> Maybe I should remove these animated gifs

They are actually quite nice. If you like to keep them I can simply replace them in the Debian with small static images.

**Comment by grahamknockillaree** [comment]
I can try tweaking the frame rate further (I already played with this to obtain the current animations).

Another possibility would be to change the relative url  of the animations to the absolute url of the animations in the gh-pages on github. I've tried that [here](https://gap-packages.github.io/hap/tutorial/chap5_mj.html#X80D0D8EB7BCD05E9) .  Do you think that might be an allowed solution?

**Comment by narc-Ontakac2** [comment]
> Do you think that might be an allowed solution?

No, that would look like a violation of privacy (which it technically is). We could track local page calls.

Don't worry, I'll patch that on the Debian side for now.

**Comment by narc-Ontakac2** [comment]
> I've uploaded two static images tutorial/images/1v2x.png and tutorial/images/1v2xcubical.png and a new file tutorial/tutorialTDA.xml. In the new .xml file you could choose to comment out the link to the animated gifs and uncomment the link to the static pngs. Does that work?

I'll check. It will probably make my life much easier.

**Comment by narc-Ontakac2** [comment]
Since I can only package releases I will wait for the next one until I use the above. Thanks.

**Comment by grahamknockillaree** [comment]
So I'll make a new release, leaving the two animated gifs in place, and you'll replace them with the two static png files in the gh-pages branch when you release the new version in Debian?

**Comment by narc-Ontakac2** [comment]
Yes, but no need to hurry with that.

**Comment by fingolfin** [comment]
What's the status of this issue?

**Comment by narc-Ontakac2** [comment]
Resolved.
